src/applications/vaos/tests/appointment-list/components/UrlAndPageTitleChange.unit.spec.js

```diff
@@ -237,123 +227,5 @@ describe('VAOS Page: AppointmentsPage', () => {
         ),
       );
     });
-
-    // WIP
-    it.skip('should show confirmed appointment detail page with new URL', async () => {
```

Comment:
I don't believe these tests are appropriate so I'm removing these WIP tests. The test should test the functionalities of `AppointmentsPage/index.jsx`. Functionalities of the sub components should be covered in their own tests.

Reply:
Sorry, just wanted to clarify here. Do you mean the test that was removed improves the test suite and should be retained? Or the change to remove the test improves the test suite?

Reply:
Good idea, I'll move them.